src/main/java/pack/service/impl/PersonService.java

```diff
@@ -0,0 +1,59 @@
+package pack.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pack.model.Person;
+import pack.repository.IPersonRepository;
+import pack.service.IPersonService;
+
+@Service
+public class PersonService implements IPersonService {
+
+	@Autowired
+	private IPersonRepository repository;
+	
+	private List<Person> personList;
+
+	@PostConstruct
+	public void init(){
+		personList=new ArrayList<>();
+	}
+
+	@Override
+	public List<Person> getPersons() {
+		try {
+			for (Person person : repository.findAll()) {
+				personList.add(person);
+			} 
+			return personList;
```

Comment:
repository.findAll() already returns a Collection of Persons. No need to add them to a list.